src/commands/update-github-release.ts

```diff
@@ -217,11 +217,11 @@ async function createNewRelease(
     prerelease: isPrerelease
   });
 
-  const success = response.status === 200;
+  const success = response.status === 201;
   if (success) {
     const releaseId = response.data.id;
 
-    await updateExistingRelease(octokit, repo, releaseId, null, null, assets);
+    return await updateExistingRelease(octokit, repo, releaseId, title, text, assets);
```

Comment:
Müsste `return` hier nicht eigentlich reichen?

```suggestion
    return updateExistingRelease(octokit, repo, releaseId, title, text, assets);
```

Reply:
Ich teste es mal.